strr-base-web/app/pages/auth/login.vue

```diff
@@ -1,14 +1,13 @@
 <script setup lang="ts">
 const { t, locale } = useI18n()
 const keycloak = useKeycloak()
-const redirectUrl = useRuntimeConfig().public.baseUrl + locale.value + '/platform/dashboard'
-const logoutUrl = useRuntimeConfig().public.baseUrl + locale.value + '/auth/login'
+const redirectUrl = useRuntimeConfig().public.baseUrl + locale.value + '/auth/login'
```

Comment:
We can remove explicitly setting the redirect url here. Keycloak will redirect to the same url it was triggered from if there's no redirect url set.